src/shaders/collision_box.fragment.glsl

```diff
@@ -6,16 +6,16 @@ void main() {
 
     float alpha = 0.5;
 
-    // Blue = collision, hide label
-    gl_FragColor = vec4(0.0, 0.0, 1.0, 1.0) * alpha;
+    // Red = collision, hide label
+    gl_FragColor = vec4(1.0, 0.0, 0.0, 1.0) * alpha;
 
-    // Black = collision, label is showing
+    // Blue = collision, label is showing
```

Comment:
Comment should say "no collision", right?

Reply:
Yep